packages/app/src/components/Admin/Security/LocalSecuritySettingContents.jsx

```diff
@@ -32,7 +32,12 @@ class LocalSecuritySettingContents extends React.Component {
 
   render() {
     const { t, adminGeneralSecurityContainer, adminLocalSecurityContainer } = this.props;
-    const { registrationMode, isPasswordResetEnabled, isEmailAuthenticationEnabled } = adminLocalSecurityContainer.state;
+    const {
+      registrationMode,
+      isPasswordResetEnabled,
+      isEmailAuthenticationEnabled,
+      isMailerSetup,
+    } = adminLocalSecurityContainer.state;
```

Comment:
`isMailerSetup` is already declared in AdminAppContainer. please use it.

Reply:
done